tutorials/descriptions.js

```diff
@@ -0,0 +1,81 @@
+module.exports.startingLinksTitles = [
```

Comment:
Again this seems to be an initial setup state of the tutorials. I am confused in why you have 6 titles and only 2 descriptions.

Also I think these "helper" files can be moved to a different directory to avoid confusion.